Entities/Items/Food/Food.as

```diff
@@ -15,7 +15,26 @@ void onInit(CBlob@ this)
 
 	this.getSprite().SetFrameIndex(index);
 	this.SetInventoryIcon(this.getSprite().getConsts().filename, index, Vec2f(16, 16));
-	this.server_setTeamNum(0); // blue fishy like in sprite sheet
+	// this.server_setTeamNum(0); // blue fishy like in sprite sheet
```

Comment:
Not sure why this is commented out now, should probably clarify why in a comment if intended